solver.rb

```diff
@@ -2,4 +2,10 @@ class Solver
   def reverse(str)
     str = str.reverse
   end
+
+  def fizzbuz(num)
+    if num % 3 == 0 && num % 5 == 0
+      'fizzbuzz'
+    end
```

Comment:
- [ ] Found this again, you created the method before writing the tests. You need to write the tests first, ensure they pass before writing the method in the solver class.

Method commit screenshot:
![a2](https://user-images.githubusercontent.com/39523312/157693427-b89bb0ed-0b17-4fb8-868f-46bd1ae1c731.png)

Test commit screenshot.
![a3](https://user-images.githubusercontent.com/39523312/157693447-b75d2f8c-0180-4651-815d-15129fa9dd9f.png)